### PR TITLE
Move unit tests to new jscodeshift test util

### PR DESCRIPTION
**Pull request by Daniel15**
Moves all the unit tests in this project to the [new jscodeshift test util](https://github.com/facebook/jscodeshift/pull/104) I added recently. This avoids the need to use the global "test" function that's being pulled in from the react-codemod project.

Unfortunately Git doesn't seem to have picked up that the test files were both moved and edited, probably because they changed so much :/

All unit tests pass on my system, with the exception of `addImportCJSBasicConfig`. That test is failing for me and it looks like some line ending issue (`\n` vs `\r\n` - I'm on Windows). It's giving me output where the expected value and actual value look identical:
```
● addImportCJSBasicConfig › it should add to a non first statement
  - Expected: '/* @flow */

          'use strict';

          var A0 = require('A0');
          var A1 = require('A1');
          var A3 = require('A3');

          aaa;' toEqual: '/* @flow */

          'use strict';

          var A0 = require('A0');
          var A1 = require('A1');
          var A3 = require('A3');

          aaa;'

```

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -29,12 +29,12 @@
     "react-codemod": "https://github.com/reactjs/react-codemod.git"
```
Comment (cpojer) [inline package.json]: you can also remove this line now.

File: transforms/__testfixtures__/touchable.output.js [hunk 2]
```diff
@@ -2,9 +2,9 @@
 
 <View>
   <TouchableBounce style={{color: 'red'}}>
-    <Image />
-    <Image />
-  </TouchableBounce>
+          <Image />
+          <Image />
+        </TouchableBounce>
```
Comment (cpojer) [inline transforms/__testfixtures__/touchable.output.js]: can you revert this change? This is a bug in recast that still hasn't been reverted :( It's ok to keep this test failing. We can also just remove it, the touchable codemod isn't relevant any more.

File: package.json (hunk 1 above)
Comment (Daniel15) [inline package.json]: Oh yeah, good point! Will do :)

File: transforms/__testfixtures__/touchable.output.js (hunk 2 above)
Comment (Daniel15) [inline transforms/__testfixtures__/touchable.output.js]: Can do. I wasn't sure if you were okay with a PR that has a failing test (even though it was failing before).

**Comment by cpojer** [comment]
Thank you so much! Can you make the same change for https://github.com/reactjs/react-codemod as well? :)

Why did you call it `defineTest`? I think `const test = require('jscodeshift').test` would have been quite nice.

**Comment by Daniel15** [comment]
> Can you make the same change for https://github.com/reactjs/react-codemod as well? :)

Yeah, I'm working on that now :+1: 

> Why did you call it defineTest? I think const test = require('jscodeshift').test would have been quite nice.

I wanted to differentiate between defining a test (which does the `define` and `it`) call, and executing the test (the actual test code itself). `test` is unclear as to which of those two it is. You can rename it if you like, though. I'm not too attached to any particular name.  :stuck_out_tongue:

File: transforms/jest-11-update.js [hunk 3]
```diff
@@ -1,7 +1,9 @@
+const path = require('path');
+
 module.exports = function(file, api, options = {}) {
   if (
-    !file.path.includes('/__tests__/') &&
-    !file.path.includes('/__mocks__/')
```
Comment (Daniel15) [inline transforms/jest-11-update.js]: This failed on Windows, since it uses `\` not `/`

**Pull request merged**